### DISCUSSION TITLE
The recent version 2.0.3 xtts works much worse

### DISCUSSION THREAD

**Discussion by dancodedev** [coqui-ai/TTS#3306] [Versions & Releases]
Good, version 2.0.3 works extremely worse than 2.0.2 in terms of audio output, is there any way to go back easily?

Has anyone noticed something similar?

http://sndup.net/fgng
http://sndup.net/j3km

**Comment by Clad3815** [comment]
Same for French

**Comment by erew123** [comment]
Please see here https://github.com/coqui-ai/TTS/discussions/3301#discussioncomment-7663904 includes reverting the model back if you dont like the way the new model sounds.

**Comment by erogol** [comment]
Also, the same answer as in the thread linked above https://github.com/coqui-ai/TTS/discussions/3301#discussioncomment-7663904

Every new version will be slightly different unless we do have the voice in the training set. 

For the samples above my vote would be 2.0.3 with better audio quality but 2.0.2 is slightly more expressive.

**Reply by dancodedev** [reply]
Version 2.0.3 has an exaggerated loss of pronunciation. There must be some error, I understand that the voices change between versions, but the loss of pronunciation and expressiveness is very great
